democracy/tests/test_hearing.py

```diff
@@ -487,59 +487,114 @@ def test_can_see_published_with_preview_code(api_client):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('geojson_fixture_name', [
-    'geojson_feature',
-    'geojson_feature_with_geometries',
+@pytest.mark.parametrize('geometry_fixture_name', [
+    'geojson_point',
```

Comment:
I guess we could add a multipoint geometry here, because that actually might sometimes be used; e.g. when hearing concerns something that is employed in many points across the city.